UPGRADING-0.5-to-0.6.md

```diff
@@ -61,7 +61,7 @@ You can stay informed about updates on the access token by listening to `client.
 
 ## Quick Troubleshooting
 
-You find yourself focussed with any of these, here are the steps to follow to upgrade your code accordingly:
```

Comment:
The variant with one s seems more common so I don't mind merging as-is, but wanted to note that this is not strictly a typo since both are valid.